### PR TITLE
Add blog post, cloud native sustainability week

### PR DESCRIPTION
**Pull request by leonardpahlke**
This PR adds the blog post outlined in #191 to the TAG ENV website.

cc @guidemetothemoon

### CODE REVIEW

**Comment by guidemetothemoon** [comment]
LGTM - thanks @leonardpahlke ! A quick heads up: I added fix for the markdown linter errors directly to the PR's branch so all the builds are green now 😊

**Comment by leonardpahlke** [comment]
Thanks ☺️

File: website/content/blog/2023-09-cloud-native-sustainability-week.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+weight: 1
+date:   2023-09-06 10:00:00 +0000
+author: Leonard Pahlke, Kristina Devochko
+---
+
+<p class="mt-5 mb-5"><img src="/images/blogs/2023-09-cloud-native-sustainability-week/banner.png"></p>
+
+The CNCF Technical Advisory Group for Environmental Sustainability (TAG ENV) is excited to announce the Cloud Native Sustainability Week 2023 and YOU are invited!
+
+Cloud Native Sustainability Week is a global event organized by the CNCF community that will take place during the second week of October (week 41). During the week, the community will come together to engage and discuss the emerging topic of environmental sustainability in the cloud native industry and open source space. In addition to the global virtual event organized by the CNCF TAG ENV, there will be 26 local meetups happening across 19 countries and 5 continents. Below, you may see a visual representation of all the local meetups that are being planned as we speak. If you would like to speak at any of the meetups, please submit your talk proposal here: <https://forms.gle/rEJfLG3a98EWJdmg7>.
```
Comment (TheFoxAtWork) [inline website/content/blog/2023-09-cloud-native-sustainability-week.md:14]:
```suggestion
Cloud Native Sustainability Week is a global event organized by the CNCF community that will take place during the second week of October (week 41). During the week, the community will come together to engage and discuss the emerging topic of environmental sustainability in the cloud native industry and open source space. In addition to the global virtual event organized by the CNCF TAG ENV, there will be 26 local meetups happening across 19 countries and 5 continents. Below, you may see a visual representation of all the local meetups that are being planned as as of this writing. If you would like to speak at any of the meetups, please submit your talk proposal here: <https://forms.gle/rEJfLG3a98EWJdmg7>.
```

**Review by TheFoxAtWork — APPROVED** [review]
One small word choice change recommended.

File: website/content/blog/2023-09-cloud-native-sustainability-week.md (hunk 1 above)
Comment (leonardpahlke) [reply website/content/blog/2023-09-cloud-native-sustainability-week.md]: 👍 updated, thanks :)

**Pull request merged**